common-practices-tools/contribution/contrib-first.md

```diff
@@ -4,19 +4,20 @@ title: Contrib First
 
 # Contrib First
 
-Whenever we are building something that could be of use by more than one project or client, we build it as contributed work first, if our contract and security concerns allow for it. Contrib First is an approach we take with both patches to FOSS software and new contributions.
+It is a best practice to consider first if we are building something that could be of use by more than one project or client. We build it as contributed work first, if our contract and security concerns allow for it. We actively advocate for this approach with our clients as we understand most are not familiar with software development. Contrib First is an approach we take with both patches to Free and Open Source Software (FOSS) and new contributions.
```

Comment:
```suggestion
It is a best practice to consider first if we are building something that could be of use by more than one project or client. We build it as contributed work first, if our contract and security concerns allow for it. We actively advocate for this approach with our clients. Contrib First is an approach we take with both patches to Free and Open Source Software (FOSS) and new contributions.
```
One of my intentions for this guidebook page is that we can use it to share the rationale with our clients.   I don't think the line about "most are not familiar with software development" is not accurate for many of our larger clients, and if it were, it is still not an uplifting thing to say.